### DISCUSSION TITLE
Is it possible to hide custom tags from intellisense?

### DISCUSSION THREAD

**Discussion by Fampov** [redhat-developer/vscode-yaml#677] [Q&A] [answered]
I work on a yaml document that is generated with a "!Script" tag in it which was always flagged as invalid when using the extension (which I love by the way). Finally I decided to add a custom tag for it so that it would no longer appear invalid. Alongside the custom tag, I've also written a custom JSON schema for these files and enabled them under the yaml.schemas which has been really helpful.

Unfortunately, now that I have created the custom tag it appears in all intellisense options alongside all my desired values from my json schema as shown in the image.
![image](https://user-images.githubusercontent.com/23152866/149414951-f87d7585-d7e2-4a66-bb27-9fd62b7e3e4d.png)

Is there any way I can hide it from the intellisense or at least order it last rather than at the top? !Script is always auto generated in the file so I never need it as an option from intellisense, I just didn't want it to keep appearing as invalid in the yaml. If I'm also using the custom tags inaccurately and there is a better way, any suggestions would be much appreciated.

Thanks for your time

**Comment by gorkem** [comment, marked as answer]
At the moment, there is no way to ignore custom tags during completion. Did you open a feature request for this?

**Reply by Fampov** [reply]
I have not but I will now. I wanted to check first in case I was overlooking something.
